documentation/FAQ.md

```diff
@@ -1,6 +1,32 @@
 Frequently Asked Questions
 ==========================
 
+* On MacOS M1 if you see any of the following errors:
```

Comment:
Is it M1 specifically? Or do you think it would be MacOS arm64 (so M1, M2, M3)

Reply:
I think `On macOS if you…` is enough since the signing issue is arch agnostic.

Reply:
I have a PR up to change this, just waiting for Mike's review, I will add you to it.

Reply:
This PR has already been merged.